internal/handlers/pg/msg_hostinfo.go

```diff
@@ -60,3 +77,26 @@ func (h *Handler) MsgHostInfo(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg
 
 	return &reply, nil
 }
+
+// ParseOSRelease parses the etc/os-release file.
+func ParseOSRelease(reader io.Reader) (string, string, error) {
```

Comment:
We should not have that function and unit test for it in different packaages.

I think the best place for both could be internal/handlers/common

Reply:
If I place them in `internal/handlers/common` and make it unexportable I cannot use it in pg package for `MsgHostInfo` func

Reply:
I placed them in `internal/handlers/common` but left the function exported. What do you think about it?